yapftests/reformatter_basic_test.py

```diff
@@ -1545,11 +1545,13 @@ def f(self, aaaaaaaaa, bbbbbbbbbbbbb, row):
     self.assertCodeEqual(code, reformatter.Reformat(uwlines))
 
   def testFunctionCallInDict(self):
+    style.SetGlobalStyle(style.CreateChromiumStyle())
```

Comment:
You're not wrong, but also, if I leave this out:

```
FAIL: testFunctionCallInDict (yapftests.reformatter_basic_test.BasicReformatterTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/brian/Code/yapf/yapftests/reformatter_basic_test.py", line 1551, in testFunctionCallInDict
    self.assertCodeEqual(code, reformatter.Reformat(uwlines))
  File "/home/brian/Code/yapf/yapftests/yapf_test_helper.py", line 57, in assertCodeEqual
    self.fail('\n'.join(msg))
AssertionError: Code format mismatch:
Expected:
 > a = {'a': b(c=d, **e)}
Actual:
 > a = {
 >     'a': b(c=d, **e)
 > }
Diff:
--- actual
+++ expected
@@ -1,3 +1 @@
-a = {
-    'a': b(c=d, **e)
-}
+a = {'a': b(c=d, **e)}
```

There's some flakiness in the test suite somewhere regarding a static reference, but rather than tracking that down, I just took this 'ensure our environment is what we think it is' approach.

Reply:
Ah! I see what's going on. When you modify the style in the tests you added, you need to do that within a try-except block. There should be some examples in the yapftests directory.

Also make sure to run `python -m yapf -r -i .` to make sure it passes the formatting checks.

Reply:
Rock on. Took care of it. Is HACKING.rst a standard I've just never seen before? That's where the "run yapf on itself to verify integrity" docs lived. Didn't even think to look there!

Reply:
The HACKING.rst is pretty bare bones as it is, but should be generally useful. It has information on how to release new versions that I follow. :-)